src/pointerwrappers.jl

```diff
@@ -1,6 +1,13 @@
 module PointerWrappers
 
-# Pointer wrapper for conveniently accessing nested structures
+export PointerWrapper
+
+"""
+    PointerWrapper(p::Ptr)
+
+Pointer wrapper for conveniently accessing nested structures. `PointerWrapper`s allow to store pointers to C types
+without the need to use `unsafe_load` even in nested structures.
```

Comment:
```suggestion
Wrap the pointer `p` to conveniently access the data fields of the underlying `struct`. This avoids having
to `unsafe_load` the entire struct when only accessing a single field, which can be helpful especially for
accessing data in nested structures.
```
Suggestion: give a little more context on the motivation for `PointerWrapper`. But feel free to change/amend as you see fit.

Reply:
Sounds good, but didn't we want to avoid "avoid" in this context (again, no pun intended)?

Reply:
Well, my intent was the following: We do not want to avoid the use of `unsafe_load` itself, but rather having to `unsafe_load` the entire struct even if we just want to access a single field. But maybe that's not clear from my description, so please feel free to improve it :-)

Reply:
LGTM, thanks @sloede.